Number_Conversions/Binary_To_Decimal/Binary_To_Decimal.java

```diff
@@ -0,0 +1,30 @@
+// Java program to convert binary to decimal 
+
+// Function to convert binary to decimal 
+class number { 
+	static int binaryToDecimal(int n) 
```

Comment:
Does this program work with this class name?

Reply:
yes

Reply:
its working